pod/perlop.pod

```diff
@@ -2875,85 +2875,151 @@ must use an C<eval()>:
+ Hello, $person!
+ And the text goes on.
+ EOT
 
-    print <<EOF;
-    The price is $Price.
-    EOF
+This yields:
 
-    print << "EOF"; # same as above
-    The price is $Price.
-    EOF
+ HELLO, JOHN!
+ AND THE TEXT GOES ON.
 
+The parentheses in the C<uc> function call don't have to be omitted:
 
-=item Single Quotes
+ print uc(<<EOT);
+ Hello, $person!
+ And the text goes on.
+ EOT
 
-Single quotes indicate the text is to be treated literally with no
-interpolation of its content.  This is similar to single quoted
-strings except that backslashes have no special meaning, with C<\\>
-being treated as two backslashes and not one as they would in every
-other quoting construct.
+ HELLO, JOHN!
+ AND THE TEXT GOES ON.
 
-Just as in the shell, a backslashed bareword following the C<<< << >>>
-means the same thing as a single-quoted string does:
+And you can intermix a here-document with other things:
 
-    $cost = <<'VISTA';  # hasta la ...
-    That'll be $10 please, ma'am.
-    VISTA
+ print <<EOT, "Followed by the next argument\n";
+ Hello, $person!
+ And the text goes on.
+ EOT
 
-    $cost = <<\VISTA;   # Same thing!
-    That'll be $10 please, ma'am.
-    VISTA
+ Hello, John!
+ And the text goes on.
+ Followed by the next argument
 
-This is the only form of quoting in perl where there is no need
-to worry about escaping content, something that code generators
-can and do make good use of.
+And you can have multiple here-documents:
 
-=item Backticks
+ print <<EOT1, <<EOT2;
+ Hello, $person!
+ And the text goes on.
+ EOT1
+ Followed by the next argument
+ EOT2
 
-The content of the here doc is treated just as it would be if the
-string were embedded in backticks.  Thus the content is interpolated
-as though it were double quoted and then executed via the shell, with
-the results of the execution returned.
+ Hello, John!
+ And the text goes on.
+ Followed by the next argument
 
-    print << `EOC`; # execute command and get results
-    echo hi there
-    EOC
+The terminator doesn't have to be a single word; it may also be some
+quoted text:
 
-=back
+ my $pagliaci = << "La Commedia e finita!";
+ A troupe comes to town to perform a play, a comedy.  The lead actress
+ and lead actor are in an unhappy marriage.  On stage, he stabs her
+ for real; then he stabs her lover who has rushed from the audience to
+ defend her.  Both die.
+ La Commedia e finita!
 
-=over 4
+There may not be a space between the C<< << >> and the identifier unless
+the terminator is quoted, as demonstrated in the example just above.
+Quoting rules for the terminator are unrelated to Perl's quoting rules.
+Only C<"">, C<''>, and C<``> can be used to quote it, NOT C<q()>,
+C<qq()>, and the like.  The only interpolation is for backslashing the
+quoting character:
+
+    print << "abc\"def";
+    testing...
+    abc"def
+
+The terminating string must appear by itself (unquoted and with no
+surrounding whitespace) on the terminating line.  And, it cannot span
```

Comment:
"And" doesn't really work as a sentence lead-in; perhaps adjoin this to the previous sentence, or replace with "also" or "finally".

Reply:
I changed `And` to `Also`